HHS/Head-Start-TTADP
# [Production] Remove "save objective" button, clear local storage when the "create new report" button is pressed

## Description of change

When "create new report" button is pressed, the report data is explicitly cleared, whereas "new" report data wasn't saved previously.

The "save objective" button on the AR form is a confusing interaction. It doesn't save, just shifts the view from form fields to a text view of the information that was just entered. Since it doesn't save, it's simple enough to remove the button and have the objectives work the same as the rest of the form.

## How to test

Start creating a new report, fill out a few fields. Do not save or navigate within the report. Return to the /activity-reports landing page. View local storage. Confirm that you have data in there. Click on "create new report" and confirm that the data is reset to the default state.

Create an AR. Check that you can save, edit, and remove an objective as before, although you no longer have the "save objective" button and the read only view that accompanies it.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-834
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-848

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
